packages/mesa.rb

```diff
@@ -29,22 +29,28 @@ class Mesa < Package
   depends_on 'libomxil_bellagio'
   depends_on 'libunwind'
   depends_on 'libvdpau'
+  depends_on 'libx11'
+  depends_on 'libxcb'
   depends_on 'libxdamage'
+  depends_on 'libxext'
+  depends_on 'libxfixes'
+  depends_on 'libxrandr'
   depends_on 'libxshmfence'
   depends_on 'libxv'
   depends_on 'libxvmc'
   depends_on 'libxxf86vm'
+  depends_on 'lm_sensors'
   depends_on 'valgrind'
   depends_on 'vulkan_headers' => :build
   depends_on 'vulkan_icd_loader'
+  depends_on 'wayland'
```

Comment:
I'm going to go out on a limb and say I imagine all these extra depends_on lines aren't needed and come along for the ride with other dependencies.  No need to list?

Reply:
I changed everything not showing up as a runtime dep to be build deps. Hopefully that should reduce installation size a little bit without breaking anything. 

We should do the same for all of the other X11 libraries, as some of the older ones may not be used by newer packages.